pkg/operator/ceph/cluster/mgr/template/prometheusrule.yaml

```diff
@@ -35,306 +35,325 @@ spec:
       record: job:ceph_versions_running:count
   - name: ceph-mgr-status
```

Comment:
@pcuzner This file is generated, right? We will need to update the source of the generation. Where can we find that generation?

Reply:
@aruniiird Do you know where the source for these rules can be found/generated?

Reply:
@pcuzner [This OpenShift enhancement](https://github.com/openshift/enhancements/pull/958) looks more like allowing the UI admin to customize alerts after they have been created. The feature in this PR is about creating customizations to include at the PrometheusRule creation time, so I see this PR as being at a different level and not expected to not conflict with the changes proposed here. The remaining question is really how we can make these customizable as proposed here without diverging from the Ceph alerts defined with the ceph-mixins. Perhaps the parameterized version of the rules used by Rook can also be used in ceph-mixins to generate the default settings for other projects.